### PR TITLE
Testnet completion checklist

### PR DESCRIPTION
**Pull request by Space-zee**

### CODE REVIEW

**Comment by Space-zee** [comment]
https://app.clickup.com/t/860q3gfx3

File: api/src/features/users/users.module.ts [hunk 1]
```diff
@@ -2,10 +2,18 @@ import { Module } from '@nestjs/common';
...
 import { User } from './user.entity';
 import { UsersService } from './users.service';
+import { UsersController } from './users.controller';
+import { TransactionsHistoryModule } from '../transactionsHistory/transactionsHistory.module';
+import { StatisticModule } from '../statistic/statistic.module';
 
 @Module({
-  imports: [TypeOrmModule.forFeature([User])],
+  imports: [
+    TypeOrmModule.forFeature([User]),
```
Comment (ruslanpalagin) [inline api/src/features/users/users.module.ts:11]: please remove user entity. it's not in use.
P.S user module was an example module

File: frontend/app/shared/header/models/userTestnetProgress.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  await waitFor(() => !!get($connectedAccountAddress));
+  await waitFor(() => !!get($usdtSmartContractPublic));
+
+  const userAddress = get($connectedAccountAddress);
+  const sc = get($usdtSmartContractPublic)
+  const balance = await sc.balanceOf(userAddress);
+  const isSupportedChain = get($isSupportedChain);
+
+  try {
+    const response: any = await httpClient.fetch(`/users/userTestnetProgress/${userAddress}`, {
```
Comment (ruslanpalagin) [inline frontend/app/shared/header/models/userTestnetProgress.ts:27]: please use apiPaths in a future

**Review by ruslanpalagin — CHANGES_REQUESTED** [review]

File: api/src/features/users/users.module.ts (hunk 1 above)
Comment (Space-zee) [reply api/src/features/users/users.module.ts:11]: +

File: frontend/app/shared/header/models/userTestnetProgress.ts (hunk 2 above)
Comment (Space-zee) [reply frontend/app/shared/header/models/userTestnetProgress.ts]: +

**Review by ruslanpalagin — APPROVED** [review]

**Pull request merged**